src/hooks/useActiveOrg.tsx

```diff
@@ -0,0 +1,61 @@
+import { getActiveOrgId, setActiveOrgId as setActiveOrgIdCookie } from "@propelauth/javascript"
+import { useContext, useEffect, useState } from "react"
+import { AuthContext } from "../AuthContext"
+
+const ORG_SELECTION_LOCAL_STORAGE_KEY = "__last_selected_org"
+const ACTIVE_ORG_ID_LOCAL_STORAGE_KEY = "__ACTIVE_ORG_ID"
+
+export function useActiveOrg() {
+    const context = useContext(AuthContext)
+    if (context === undefined) {
+        throw new Error("useActiveOrg must be used within an AuthProvider or RequiredAuthProvider")
+    }
+    const [activeOrgIdState, setActiveOrgIdState] = useState<string | undefined>(getActiveOrgId())
```

Comment:
If we have the state in a cookie and localStorage, do we also need React state for it? We could just treat the cookie as the source of truth and localStorage as a useful way to get notified on changes?

Reply:
We still need something that listens to the local storage changes, and the cookie cannot do that, nor should do that, as it is the source of truth. If the localStorage is just a timestamp anyways, then the state is the one that needs to get updated with the cookies value, and show what is returned.